README.md

```diff
@@ -79,13 +80,19 @@ Snapshots of the development version are available in [Sonatype's snapshots repo
 The configuration for an Android project is very similar to the Java case, with two key differences:
 
 1. The `net.ltgt.apt` plugin is not required.
-2. Rather than declaring NullAway as an `apt` dependence, use an `annotationProcessor` dependence:
+2. The `com.google.code.findbugs:jsr305:3.0.2` dependence can be removed; you can use the `android.support.annotation.Nullable` annotation from the Android Support library.
+3. Rather than declaring NullAway as an `apt` dependence, use an `annotationProcessor` dependence:
 
 ```gradle
 dependencies {
   annotationProcessor "com.uber.nullaway:nullaway:0.2.2"
 }
 ```
+See the [sample-app `build.gradle`](https://github.com/uber/NullAway/blob/master/sample-app/build.gradle) for a complete example.
+
+#### Annotation Processors / Generated Code
+
+Some annotation processors like [Dagger](https://google.github.io/dagger/) and [AutoValue](https://github.com/google/auto/tree/master/value) generate code into the same package namespace as your own code.  This can cause problems when setting NullAway to the `ERROR` level as suggested above, since errors in this generated code will block the build.  Currently the best solution to this problem is to completely disable Error Prone on generated code, using the `-XepExcludedPaths` option added in Error Prone 2.13 (documented [here](http://errorprone.info/docs/flags)).  To use, figure out which directory contains the generated code, and add that directory to the excluded path regex.
```

Comment:
Nit - I wonder if a more generally applicable one would be to not touch anything annotated with `@Generated`. Dagger and most auto-stuff includes this annotation in generated code

Reply:
There exists something like that already in EP

```
-XepDisableWarningsInGeneratedCode : Disables warnings in classes annotated with @javax.annotation.Generated
```

This is more for processors or other tools that generate code without that annotation

Reply:
This flag is also slightly different because the `-XepDisableWarningsInGeneratedCode` will still run on the generated code and not report anything.

While the `-XepExcludedPaths` will not even run EP on the code that is excluded which is beneficial for build times.

Reply:
We shouldn't mention dagger then since it does generate that annotation right?

Reply:
`-XepDisableWarningsInGeneratedCode` is not set by default. I think we can message with something like

"There are two flags to disable running on generated code" and direct them to EP docs rather than us being too specific

Reply:
The key is that `-XepDisableWarningsInGeneratedCode` only disables *warnings*.  When you set NullAway to ERROR level then NullAway issues in Dagger-generated code will still break the build.  Plus, there are annotation processors that don't add `@Generated`.  I really think `-XepExcludedPaths` is the best solution here, and would rather not mention `-XepDisableWarningsInGeneratedCode`.